src/styles/20-tools/_mixins/_link.scss

```diff
@@ -0,0 +1,110 @@
+@mixin link-base {
```

Comment:
File is a combination of the following from corporate-components:

Placeholders from [extends > typography](https://github.com/wellcometrust/corporate-components/blob/develop/src/assets/styles/20-tools/_extends/_typography.scss) repurposed as mixins and `%anchor-` prefixes in names replaced with `link-`

Mixins from [mixins > typography](https://github.com/wellcometrust/corporate-components/blob/develop/src/assets/styles/20-tools/_mixins/_typography.scss)